seqseek/chromosome.py

```diff
@@ -27,7 +28,12 @@ def __init__(self, chromosome_name, assembly=BUILD37, loop=False):
         """
         self.name = str(chromosome_name)
         self.assembly = assembly
-        self.loop = loop
+        if loop is None:
+            self.loop = True if self.name in MITOCHONDRIA_NAMES else False
+            self.RCRS_N_remove = RCRS_N_remove
+        else:
+            self.loop = loop
+            self.RCRS_N_remove = False
```

Comment:
if someone has explicitly requested the N to be included it seems counter-intuitive to silently remove it if `loop` was also specified. Even when loop is specified, SeqSeek won't return sequences longer than the length of the mitochondria so you don't have to worry about multiple `N` bases appearing in the returned sequence.

Reply:
Reverted to original, but added RCRS_N_remove optional parameter, defaults to True to retain original behavior